invenio/modules/access/local_config.py

```diff
@@ -24,12 +24,14 @@
 
 # pylint: disable=C0301
 
-from invenio.config import CFG_SITE_NAME, CFG_SITE_URL, CFG_SITE_SECURE_URL, CFG_SITE_SUPPORT_EMAIL, CFG_SITE_RECORD, CFG_SITE_ADMIN_EMAIL
 from invenio.base.i18n import _
-from invenio.base.globals import cfg as config
+from invenio.base.config import CFG_SITE_URL, CFG_SITE_RECORD, \
```

Comment:
Can you check where these variables are used and maybe we can remove them. Alternatively you can check `lazy_string` (from `speaklater` library).